src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateTFMRule.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Text;
+using System.Globalization;
+using Microsoft.Build.Construction;
+using System.Linq;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    // TODO: Support Multi-TFM
+    public class MigrateTFMRule : IMigrationRule
+    {
+        private readonly ITransformApplicator _transformApplicator;
+        private readonly AddPropertyTransform<NuGetFramework>[] _transforms;
+
+        public MigrateTFMRule(TransformApplicator transformApplicator = null)
```

Comment:
I don't ever see the "applicator" parameter being specified when calling these constructors.  Can they be removed?

A reason I mention it here is because this constructor takes in the concrete class, and not the interface.